sematic/ui/cypress/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Having this file makes `vscode` happy. (intellisense). But I doubt cypress CLI cares about it.